app/controllers/catalog_controller.rb

```diff
@@ -2,7 +2,6 @@ class CatalogController < ApplicationController
   include Hydra::Catalog
   include Hydra::Controller::ControllerBehavior
   include Sufia::Catalog
-  include BlacklightAdvancedSearch::Controller
 
```

Comment:
sounds good, we can always add it back in if we want/need it. I'm down with keeping things as simple as we can get away with for soft launch.